rest-server/bin/engines.py

```diff
@@ -25,7 +25,7 @@
     # print("Engine-id: " + engine_id)
     # print("Json config: \n" + str(config))
     try:
-        res = engine_client.update(engine_id=engine_id, data=config)
+        res = engine_client.update(engine_id=engine_id, input_path=args.input, update_config=True, data=config)
```

Comment:
the API to engine_client.update(...) has changed to support 2 variants, specified by the query part of the URL